planner/core/planbuilder.go

```diff
@@ -217,9 +217,15 @@ func (b *PlanBuilder) Build(node ast.Node) (Plan, error) {
 		return b.buildSimple(node.(ast.StmtNode))
 	case ast.DDLNode:
 		return b.buildDDL(x)
+	case *ast.ChangeStmt:
+		return b.buildChange(x)
 	}
 	return nil, ErrUnsupportedType.GenWithStack("Unsupported type %T", node)
 }
+func (b *PlanBuilder) buildChange(v *ast.ChangeStmt) (Plan, error) {
```

Comment:
Add a blank line in front of this function.

Reply:
ok, i already add a blank line in front of this function.